service/src/test/java/bio/terra/workspace/common/GcpCloudUtils.java

```diff
@@ -195,7 +196,7 @@ private static Storage getGcpStorageClient(GoogleCredentials userCredential, Str
   }
   /**
    * Get a result from a call that might throw an exception. Treat the exception as retryable, sleep
-   * for 15 seconds, and retry up to 40 times. This structure is useful for situations where we are
```

Comment:
if 60 times work, would you mind update here too thx :)  https://cs.github.com/DataBiosphere/terra-workspace-manager/blob/3bd1f03cd912ea8fe93e5678a7b97876df71f1a4/integration/src/main/java/scripts/utils/ClientTestUtils.java?q=ClientTestUtils+#L270